OpenRA.Mods.Common/Terrain/DefaultTileCache.cs

```diff
@@ -13,10 +13,11 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using OpenRA.Graphics;
 using OpenRA.Primitives;
 using OpenRA.Support;
 
-namespace OpenRA.Graphics
+namespace OpenRA.Mods.Common.Terrain
 {
 	class TheaterTemplate
```

Comment:
```suggestion
	public class TheaterTemplate
```

to avoid duplication?